about.html

```diff
@@ -0,0 +1,120 @@
+        <img src="images/close.png" id="close-menu" onclick="display()" alt="Close Icon">
+      </div>
+      <div class="mob-nav-container" id="menu">
+        <ul class="mob-nav-ul">
+          <li>
+            <a href="index.html" onclick="display()">Home</a>
+          </li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+        </ul>
+      </div>
+      <div class="about-landing-text">
+        <h4>"Live Today, Love Tomorrow, Until Forever"</h4>
+        <h1>Tomorrowland <br> world tour concert 2022</h1>
+        <div class="middle-text">
+          <p>THE PEOPLE OF TOMORROW<br>
+            We believe...<br>
+            in a commitment to create a reality that relates positively with Mother Nature and contributes to the well-being of the next generations. Let’s engage with each other and do something good today, for which we will be grateful for tomorrow.</p>
+        </div>
+      </div>
+      <div class="about-date-place">
+        <h2>Please Contact us per Email for any further question about Tomorrowland world tour concert 2022</h2>
+        <h3>Tomorrowland2022@cckorea.org</h3>
+      </div>
+    </section>
+    <section class="logo-section">
+      <div class="about-featured-head"> 
+        <h2>Tomorrowland world tour concert 2022 Logo</h2>
+        <hr class="underline">
+      </div>
+      <div class="logo-disc">
+        <h2>Please Contact us per Email for any further question about Tomorrowland world tour concert 2022</h2>
+      </div>
+      <div class="logo-img">
+        <img src="images/footer-img.png" alt="Concert Logo">
+      </div>
+      <hr>
+      <div class="about-featured-head"> 
+        <h2>See the past Tomorrowland world tour concert</h2>
+        <hr class="underline">
+      </div>
+      <div class="logo-disc">
+        <h2>Please Contact us per Email for any further question about Tomorrowland world tour concert 2022</h2>
+      </div>
+      <div class="past-wrapper">
+        <div class="past-container">
+          <div class="past-image1">
+            <div class="past-image-wrapper">
+            </div>
+          </div>
+          <div class="past-image-disc">
+            <h2>2019</h2>
+            <h3>CC Globbal Summit 2020</h3>
+          </div>
+        </div>
+        <div class="past-container">
+          <div class="past-image2">
+            <div class="past-image-wrapper">
+            </div>
+          </div>
+          <div class="past-image-disc">
+            <h2>2020</h2>
+            <h3>CC Globbal Summit 2020</h3>
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer class="about-footer-section">
+    <div class="about-footer-wrappter">
+        <img src="images/footer-img.png" alt="creative-common">
+        <p>2022 Tomorrowland world tour Rights Reserved</p>
+    </div>
+  </footer>
```

Comment:
- [ ] Your about footer is still quite not similar to the template in the mobile version. Kindly align the logo to the left and the text to the right and resize your logo.
Your design:
<img width="499" alt="image" src="https://user-images.githubusercontent.com/61919617/181103217-8c5dde3c-179a-4b5d-91f0-de4dcd092a58.png">
Template design:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/61919617/181103315-c9316aed-e421-434e-a097-e2de15637db0.png">
<img width="418" alt="image" src="https://user-images.githubusercontent.com/61919617/181103316-37f0d9f1-9086-4f90-b582-d46c325e0eee.png">